src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.ConflictResolution.targets

```diff
@@ -22,9 +22,13 @@ Copyright (c) .NET Foundation. All rights reserved.
     
     <ItemGroup>
       <!-- We need to find all the files that will be loaded from deps for conflict resolution.
-          To do this, we look at the files that would be copied local when CopyLocalLockFileAssemblies is true
+          To do this, we look at the files that would be copied local when CopyLocalLockFileAssemblies is true.
+          However, if CopyLocalLockFileAssemblies is true, then we don't add these items, as they
+          will always be included in ReferenceCopyLocalPaths.
           -->
-      <_LockFileAssemblies Include="@(AllCopyLocalItems->WithMetadataValue('Type', 'assembly'))" />
+      <_LockFileAssemblies Include="@(AllCopyLocalItems->WithMetadataValue('Type', 'assembly'))"
```

Comment:
Today, CopyLocalLockFileAssemblies for netcoreapp include shared framework and don't respect other publish filtering mechanisms either. I think that's what you want here, but I also want to fix CopyLocalLockFileAssemblies to be more like "publish in place". See #933

I'm annoyed by the fact that the log can be filled with items called *CopyLocal* that are not in fact copied anywhere. I'm not pushing back on this change because we're already in that boat, but it's something that's been on my mind for a while.